docs/resources/nic.md

```diff
@@ -27,6 +27,13 @@ resource "ionoscloud_ipblock" "example" {
     name                = "IP Block Example"
 }
 
+resource "ionoscloud_lan" "example"{
```

Comment:
best make this a separate example that uses ipv6

Reply:
ipv6 example added separately